### PR TITLE
defect #731006: Fix entity label icons for search side

### PR DESCRIPTION
**Pull request by pikachugb**

### CODE REVIEW

File: octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/search/SearchEditor.java [hunk 1]
```diff
@@ -149,6 +149,7 @@ public void done(IJobChangeEvent event) {
                         if (entityData.getOriginalEntityList().size() == 0) {
                             container.showControl(noSearchResultsComposite);
                         } else {
+                        	entityListComposite.refreshIcons();
```
Comment (atothhpe) [inline octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/search/SearchEditor.java:152]: weird spacing in PR

**Review by atothhpe — COMMENTED** [review]
whitespace?

**Review by faraverete — APPROVED** [review]
ctrl alt L / ctrl alt O. But man what a beautiful code

**Comment by faraverete** [comment]
I'm Vlad, wrong account :)

**Pull request merged**